target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4018-ea6350v3.dts

```diff
@@ -0,0 +1,287 @@
+/* Copyright (c) 2015, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2018, Oceanic Systems (UK) Ltd. All rights reserved.
+ * Copyright (c) 2018, Oever Gonzalez. All rights reserved.
+ *
+ * This file is based off of the qcom-ipq4018-** device files as attributed 
+ * to the Linux Foundation. 
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "Linksys EA6350v3 based on Qualcomm Technologies, Inc. IPQ40xx/AP-DK01.1-C2";
+	compatible = "linksys,ea6350v3", "qcom,ipq4019";
```

Comment:
Note: @robimarko recently made a update for the ipq40xx for the upcoming switch to 4.19.
As a result the dts in files-4.14/arch/... need to be duplicated into files-4.19/arch/... for the time being.
Do you think you can also add files-4.19/arch/arm/boot/dts/qcom-ipq4018-ea6350v3.dts to your PR?

(And if you do, for the 4.19 series please remove the "qcom,ipq4019" here as it is no longer necessary)